py/desispec/workflow/procfuncs.py

```diff
@@ -239,7 +239,7 @@ def create_and_submit(prow, queue='realtime', reservation=None, dry_run=0, joint
         prow['SCRIPTNAME'] = orig_prow['SCRIPTNAME']
     return prow
 
-def desi_proc_command(prow, system_name, use_specter, queue=None):
+def desi_proc_command(prow, system_name, use_specter=False, queue=None):
```

Comment:
Do we want/need to propagate `no_gpu` here?

Reply:
Documenting the reasoning for not including no_gpu here:  desi_proc_command is only used by higher level pipeline wrappers (create_batch_script called by desi_run_night and desi_daily_proc_manager) and those higher level scripts don't provide `--no-gpu` either.  They either auto-derive which jobs should be sent to GPU vs. CPU nodes, or use `--system-name` to send all jobs to one or the other.  `--no-gpu` is provided to lower level scripts like desi_proc just in case, but even then it is questionable why you would want to be on a GPU node but not use the GPU, vs. just doing the same testing on a CPU node.

If you really really need to be on a GPU node but not use the GPUs, you can also set $DESI_NO_GPU for a global opt-out, and if that becomes a common use-case then we can add a `--no-gpu` option at a higher level in the pipeline options, but in the meantime I haven't added and option that I don't think we'll use.